docs/framework/wcf/samples/mtom-encoding.md

```diff
@@ -64,7 +64,7 @@ Press <ENTER> to terminate client.
   
 1. Install ASP.NET 4.0 using the following command.  
```

Comment:
I have commented on other places as well.  Please update this to something else other than running aspnet_regiis.exe.  Maybe simply say "enable ASP.NET"?